hphp/runtime/vm/jit/vasm-instr.h

```diff
@@ -1101,6 +1105,10 @@ struct shlq { Vreg64 s, d; VregSF sf; Vflags fl; }; // uses rcx
 /*
  * arm intrinsics.
  */
+struct csoneb { ConditionCode cc; VregSF sf; Vreg8 t, d; };
```

Comment:
I just split out the ARM-specific simplifications into a separate file (in 16710e2), so you can move all your new simplifications there and just define `csinc{}`.